README.md

```diff
@@ -1 +1,67 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+
+This project is to set up github supository and set up linters on a portfolio project.
+The project is in a nub=mber of sections wich inclusde the headers, the name and title, the introduction and my abilities sections
+
+## Built With
+
+- html
+- node
+
+
+## Live Demo (if available)
+
+no demo
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [OPTIONAL] Kindly try to be more specific about the steps the user needs to make to use the app and delete the sections that are not important for this app. It is not clear for a new user how to start the application, what technologies it uses, etc.